plugins/core/IO/qCoreIO/src/ObjFilter.cpp

```diff
@@ -145,14 +146,26 @@ CC_FILE_ERROR ObjFilter::saveToFile(ccHObject* entity, const QString& filename,
 		if (withTriNormals)
 		{
 			NormsIndexesTableType* normsTable = mesh->getTriNormsTable();
+
+			//reset save dialog
+			unsigned numTriangleNormals = normsTable->currentSize();
+			if (parameters.parentWidget)
```

Comment:
can you test 'pDlg' directly? (instead of parameters.parentWidget). Seems more straightforward.